src/python/pants/backend/scala/target_types.py

```diff
@@ -359,10 +376,187 @@ class ScalacPluginTarget(Target):
     )
 
 
+# -----------------------------------------------------------------------------------------------
+# `scala_artifact` target
+# -----------------------------------------------------------------------------------------------
+
+
+class ScalaArtifactGroupField(JvmArtifactGroupField):
+    pass
+
+
+class ScalaArtifactArtifactField(JvmArtifactArtifactField):
+    pass
+
+
+class ScalaArtifactVersionField(JvmArtifactVersionField):
+    pass
+
+
+class ScalaArtifactResolveField(JvmArtifactResolveField):
+    pass
+
+
+class ScalaArtifactPackagesField(JvmArtifactPackagesField):
+    pass
```

Comment:
It shouldn't be necessary to subclass these unless you'd like them to have different behavior or a different docstring.